<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1405</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dormitory</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="room2">
       <attribute name="title">
        <string>방(1학년)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="2">
           <widget class="QPushButton" name="btn_1g">
            <property name="text">
             <string>배치</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QTextBrowser" name="lbl_1g">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="check_1g">
            <property name="text">
             <string>현재 배치 저장</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="room3">
       <attribute name="title">
        <string>방(2학년)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="2">
           <widget class="QPushButton" name="btn_2g">
            <property name="text">
             <string>배치</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QTextBrowser" name="lbl_2g">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="check_2g">
            <property name="text">
             <string>현재 배치 저장</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="room4">
       <attribute name="title">
        <string>방(3학년)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="lbl_3g">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btn_3g">
            <property name="text">
             <string>배치</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="check_3g">
            <property name="text">
             <string>현재 배치 저장</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="studyM">
       <attribute name="title">
        <string>자습실(남)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="2">
           <widget class="QPushButton" name="btn_sm">
            <property name="text">
             <string>배치</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="check_sm">
            <property name="text">
             <string>현재 배치 저장</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QTextBrowser" name="lbl_sm">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="studyF">
       <attribute name="title">
        <string>자습실(여)</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_12">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="lbl_sf">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="check_sf">
            <property name="text">
             <string>현재 배치 저장</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btn_sf">
            <property name="text">
             <string>배치</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="current">
       <attribute name="title">
        <string>현재 배치</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QTextBrowser" name="lbl_c">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btn_xlsx">
            <property name="text">
             <string>내보내기(.xlsx)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="btn_txt">
            <property name="text">
             <string>내보내기(.txt)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="rBtn_floor">
            <property name="text">
             <string>층별 저장</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QRadioButton" name="rBtn_grade">
            <property name="text">
             <string>학년별 저장</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="map">
       <attribute name="title">
        <string>도면</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="1" column="4">
           <widget class="QPushButton" name="map_1">
            <property name="text">
             <string>기숙사(2층)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="pic">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QPushButton" name="map_2">
            <property name="text">
             <string>기숙사(3층)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="4">
           <widget class="QPushButton" name="map_3">
            <property name="text">
             <string>기숙사(4층)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1405</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionUpload"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>종료</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionstudent">
   <property name="text">
    <string>학생 파일</string>
   </property>
  </action>
  <action name="actionStructure_File">
   <property name="text">
    <string>도면 파일</string>
   </property>
  </action>
  <action name="action_xlsx_file">
   <property name="text">
    <string>.xlsx file</string>
   </property>
  </action>
  <action name="action_png_file">
   <property name="text">
    <string>.png file</string>
   </property>
  </action>
  <action name="actionVisual">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>시각화 모드</string>
   </property>
  </action>
  <action name="actionList">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>목록 모드</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>도움말 바로가기</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/user/.designer/backup/pic/upload.png</normaloff>C:/Users/user/.designer/backup/pic/upload.png</iconset>
   </property>
   <property name="text">
    <string>파일 업로드</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
